packages/runtime/src/store/tutorial-runner.ts

```diff
@@ -189,6 +189,62 @@ export class TutorialRunner {
     );
   }
 
+  async fileExists(filepath: string) {
+    const previousLoadPromise = this._currentLoadTask?.promise;
+
+    return new Promise<boolean>((resolve, reject) => {
+      this._currentLoadTask = newTask(
+        async (signal) => {
+          await previousLoadPromise;
+
+          const webcontainer = await this._webcontainer;
+
+          if (signal.aborted) {
+            reject(new Error('Task was aborted'));
+          }
+
+          signal.throwIfAborted();
```

Comment:
@Nemikolh, extra attention here please 😄 
Is this correct?

Reply:
It is! :raised_hands: 

Thinking more about this, given what this method does, I think that we should just remove everything related to cancellation here. It's mostly there to speed up when moving between lessons quickly.